internal/kafka/command_link_describe.go

```diff
@@ -61,6 +65,11 @@ func newDescribeLink(link kafkarestv3.ListLinksResponseData, topic string) *link
 	if link.GetLinkError() != "NO_ERROR" {
 		linkError = link.GetLinkError()
 	}
+	linkCategories := link.GetCategoryCounts()
+	categories := make([]string, len(linkCategories))
+	for i, category := range linkCategories {
+		categories[i] = fmt.Sprintf(`%s: %d`, category.StateCategory, category.Count)
```

Comment:
nit: which one is better?

This could be too verbose after string concatenation:
`categories[i] = fmt.Sprintf(`%s count: %d`, category.StateCategory, category.Count)`

This could lead to unclear representation of the number:
`categories[i] = fmt.Sprintf(`%s: %d`, category.StateCategory, category.Count)`

Reply:
Maybe we can change the table definition to below and use option#2 above?

CategoryCounts string `human:"Mirror Partition States Count,omitempty" serialized:"category_counts,omitempty"`

Reply:
Good point, I would say add "Count" for table definition is better, It may cause confusion for clients without it.
fyi: the alignment https://confluentinc.atlassian.net/wiki/spaces/K/pages/3620274183/Mini+1-Pager+-+Update+description+for+Cluster+Linking#Kafka-CLI